.devcontainer/Dockerfile

```diff
@@ -33,3 +34,28 @@ ENV DOCKER_BUILDKIT=1
 
 # Add daprd to the path for the VS Code Dapr extension.
 ENV PATH=$PATH:/home/vscode/.dapr/bin
+
+
+# Proxy
+ARG PROXY
+ENV PROXY="${PROXY}"
+
+ARG PROXY_HOST
+ENV PROXY_HOST="${PROXY_HOST:-http://host.docker.internal}"
+
+ARG PROXY_PORT
+ENV PROXY_PORT="${PROXY_PORT:-3128}"
```

Comment:
It's unclear to me why we need default proxy configuration here. Proxy config should be always env dependent, or?